### PR TITLE
Reduce deploy fee

### PR DESCRIPTION
**Pull request by Tommo-L**
Close https://github.com/neo-project/neo/issues/2151

Some discussion 
https://github.com/neo-project/neo/pull/2140#issuecomment-745388758
https://github.com/neo-project/neo/issues/2001

### CODE REVIEW

**Comment by Tommo-L** [comment]
In the future, we can raise it appropriately, and the threshold should be lowered in the early stage.

File: src/neo/SmartContract/Native/ManagementContract.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ private int GetNextAvailableId(StoreView snapshot)
 
         internal override void Initialize(ApplicationEngine engine)
         {
-            engine.Snapshot.Storages.Add(CreateStorageKey(Prefix_MinimumDeploymentFee), new StorageItem(100_00000000));
+            engine.Snapshot.Storages.Add(CreateStorageKey(Prefix_MinimumDeploymentFee), new StorageItem(1_00000000));
```
Comment (erikzhang) [inline src/neo/SmartContract/Native/ManagementContract.cs:34]: 10GAS?

Comment (Tommo-L) [reply src/neo/SmartContract/Native/ManagementContract.cs]: Done. It's good for me, better than 100GAS.

**Review by erikzhang — APPROVED** [review]
Let's wait for other developers' comments. @neo-project/core @neo-project/ngd-seattle @neo-project/ngd-shanghai @roman-khimov

**Comment by devhawk** [comment]
> In the future, we can raise it appropriately, and the threshold should be lowered in the early stage.

Should it be configurable? (in the future, not for preview 4)

**Comment by superboyiii** [comment]
I agree with this. 
>Should it be configurable? (in the future, not for preview 4)
Yes, it should be configurable, after preview4.

**Comment by Tommo-L** [comment]
> Should it be configurable? (in the future, not for preview 4)

@devhawk  @superboyiii  It's already configurable. 

https://github.com/neo-project/neo/blob/06d4345d430a6b31ec651d0495182dcc7c4114f7/src/neo/SmartContract/Native/ManagementContract.cs#L60-L66

**Comment by superboyiii** [comment]
> Prefix_MinimumDeploymentFee

Got it.

**Review by roman-khimov — APPROVED** [review]
10 is good for me.

**Review by superboyiii — APPROVED** [review]

**Review by steven1227 — APPROVED** [review]
Support on it.

**Review by shargon — APPROVED** [review]

**Comment by superboyiii** [comment]
Merge?

**Comment by superboyiii** [comment]
We could wait for a bit more hours for others. If no disagreement, I think it could be merged.

**Review by vncoelho — APPROVED** [review]

**Pull request merged**